File: ejercicio3-camara-microfono-ios/CamaraMicrofonoIOS/Models/AppTheme.swift
import SwiftUI

/// Same two brand themes as Ejercicio 2, kept identical on purpose so both
/// apps look consistent.
enum AppTheme: String, CaseIterable, Identifiable, Codable {
    case guinda
    case azul

    var id: String { rawValue }

    var label: String {
        switch self {
        case .guinda: return "Guinda (IPN)"
        case .azul: return "Azul (ESCOM)"
        }
    }

    var accentColor: Color {
        switch self {
        case .guinda: return Color(red: 0.549, green: 0.114, blue: 0.251) // #8C1D40
        case .azul: return Color(red: 0.0, green: 0.341, blue: 0.718)    // #0057B7
        }
    }
}

File: ejercicio5-kmp-camara-microfono/CamaraMicKMP/shared/src/commonMain/kotlin/mx/ipn/escom/camaramic/AppTheme.kt
package mx.ipn.escom.camaramic

// Shared brand colors (as 0xAARRGGBB longs) so both the Compose (Android)
// and SwiftUI (iOS) UIs use the exact same Guinda/Azul values.
enum class AppThemeOption(val label: String, val argb: Long) {
    GUINDA("Guinda (IPN)", 0xFF8C1D40),
    AZUL("Azul (ESCOM)", 0xFF0057B7),
}
